### DISCUSSION TITLE
[Feature Request]: Remove "Arizona - New Mexico Chapter" from SEINet banner

### DISCUSSION THREAD

**Discussion by MiguelGrande** [Symbiota/community-board#669] [Wish List]
### Symbiota Portal Name

SEINet

### Is your feature/tool request related to a problem? Please describe.

I am puzzled why "Arizona - New Mexico Chapter" remains on the SEINet banner. This has always been confusing to me, and I presume is there now because of history, or someone doesn't want to let it go. As I see it, SEINet is an umbrella for a number of portals, including "Arizona - New Mexico Chapter" but many more. I suggest removing it from the banner. Another thing that might be done is to have a new meaning for SEINet. What did it mean in the past? What does it mean now?  How about something like Symbiota Environmental Information Network (maybe something better for the "E"). 

### Describe the solution you'd like.

As above, simply remove "Arizona - New Mexico Chapter" from the SEINet banner, possibly consider/come up with a meaning for the SEINet acronym.

**Comment by themerekat** [comment]
Hi @MiguelGrande , we updated the homepage text to explain a little more about SEINet. You're right that it no longer functions as only the AZ-NM chapter, but there is no central "SEINet" portal, in reality!

**Reply by MiguelGrande** [reply]
Right, understood.  SEINet is an umbrella/hub for many portals, yes?  In any case, I still think removing the "Arizona - New Mexico Chapter" (only one of several portals in SEINet) from the banner a good idea; we'll see if others feel the same...

**Reply by themerekat** [reply]
Yes, but the Arizona-New Mexico folks are a particular community of users who would no longer have a "home" portal if we rebranded theirs to be generic SEINet. We've been talking about this a lot, but we haven't come to a solution that would satisfy everyone, yet.

**Comment by MiguelGrande** [comment]
Oh, I see.  Even though their portal is listed as one of many, it’s really not separate from SEINet, correct?  And, that’s important to them?  

What would it take to make the Arizona - New Mexico Chapter a separate portal, like this one?:
https://midwestherbaria.org/portal/
and just call it that, Arizona - New Mexico Chapter?  still part of SEINet?  

-M

**Reply by themerekat** [reply]
We're not entirely sure, tbh. Making a separate "superportal" front-end for the larger "SEINet" entity is one of the options, but we're a little hesitant to make new portals to access the same data...we'd prefer to decrease the number of portals, given the time it takes to update our current 50-or-so portals to the latest codebase (it has to be done for every front end).

**Comment by MiguelGrande** [comment]
I just checked https://midwestherbaria.org/portal/index.php. Aside from a separate homepage, the Search function seems identical to SEINet now; it lists all the portals.  

I don’t know the ins and outs of this like you do, but seems it’d be an easy fix, make just kind of a facade for the Arizona - New Mexico Chapter portal, just one new website.  But, of course, I could be missing something.  

-M

**Reply by themerekat** [reply]
Right, the database is the same, but it is an entirely new "portal" with a new copy of the Symbiota code, which requires periodic updating = more maintenance

**Comment by MiguelGrande** [comment]
Sorry to go around again.  You know much more about this than I do.  But, is it possible/feasible just to have another (duplicate) Home page for the Arizona - New Mexico Chapter, but essentially the same as the SEINet Home page (why I used the term facade), just a different banner and introductory material about the chapter?  Not have a new copy of the Symbiota code?  So, maintenance would be minimal to none?  -Mike

**Reply by themerekat** [reply]
I don't _think_ so, since all the different front ends are copies of the code, but I'm not the expert on what is possible in terms of new pages.
